components/atoms/ActionButton.js

```diff
@@ -22,7 +22,7 @@ export function ActionButton(props) {
         props.disabled
           ? "bg-gray-light text-gray-600 border border-gray-md"
           : props.className
-      } text-sm px-4`}
+      } text-base font-body font-bold`}
```

Comment:
Can we remove "font-bold" from the buttons?

Reply:
Removed font-bold from action button